### PR TITLE
HDInsight spec changes

### PR DESCRIPTION
**Pull request by shefaliv**
This includes doc changes + changes made to the HDI specs while setting up the initial .NET SDK project.

This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [ ] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [ ] My spec meets the review criteria:
  - [ ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by jhendrixMSFT** [comment]
I'm reviewing this today.

File: arm-hdinsight/2015-03-01-preview/swagger/cluster.json [hunk 1]
```diff
@@ -96,62 +96,12 @@
                 },
                 "x-ms-long-running-operation": true
             },
-            "patch": {
```
Comment (jhendrixMSFT) [inline arm-hdinsight/2015-03-01-preview/swagger/cluster.json]: Why was this operation removed?

File: arm-hdinsight/2015-03-01-preview/swagger/cluster.json [hunk 2]
```diff
@@ -96,62 +96,12 @@
...
             },
-            "patch": {
-                "tags": [
-                    "Clusters"
-                ],
-                "operationId": "Clusters_Update",
-                "description": "Patch HDInsight cluster with the specified parameters.",
-                "x-ms-examples": {
-                    "Patch HDInsight Linux clusters": {
-                        "$ref": "../examples/PatchLinuxHadoopCluster.json"
```
Comment (jhendrixMSFT) [inline arm-hdinsight/2015-03-01-preview/swagger/cluster.json]: If this operation is really no longer required then you should delete the corresponding example (I assume it's no longer required).

File: arm-hdinsight/2015-03-01-preview/swagger/cluster.json (hunk 1 above)
Comment (shefaliv) [inline arm-hdinsight/2015-03-01-preview/swagger/cluster.json]: We don't actually surface any such API currently. Not sure why it was here in the first place.

File: arm-hdinsight/2015-03-01-preview/swagger/cluster.json (hunk 2 above)
Comment (shefaliv) [inline arm-hdinsight/2015-03-01-preview/swagger/cluster.json]: Makes sense.

**Comment by jhendrixMSFT** [comment]
Please take a look at the model validation errors here.  https://travis-ci.org/Azure/azure-rest-api-specs/jobs/243796372  Your changes from string to boolean require matching updates in the example files.

**Review by salameer — CHANGES_REQUESTED** [review]
Please review requested changes

**Comment by salameer** [comment]
@shefaliv kindly reply to the comments :)

We will be closing this request if there's no response by end of week

Thanks,

Samer

**Comment by salameer** [comment]
Due to no reponses @shefaliv  this PR will be closed, please re submit when your ready with the requested changes

**Pull request closed without merging**